src/schemas/FEEDBACK-TOOL/schema.js

```diff
@@ -1,407 +1,416 @@
-import constants from '../../common/constants';
+import definitions from "../../common/definitions";
+import schemaHelpers from "../../common/schema-helpers";
+import constants from "../../common/constants";
 
 const { salesforceCountries: countries } = constants;
 
 // The common definition includes "II" and lacks "Other"
-const fullName = _.set('properties.suffix', {
-  type: 'string',
-  'enum': [
-    'Jr.',
-    'III',
-    'IV',
-    'Sr.',
-    'Other'
-  ]
-}, definitions.fullName);
+const fullName = _.set(
+  "properties.suffix",
+  {
+    type: "string",
+    enum: ["Jr.", "III", "IV", "Sr.", "Other"]
+  },
+  definitions.fullName
+);
 
-const USA = countries.find(country => country.value === 'USA');
-const nonUSACountries = countries.filter(country => country.value !== 'USA');
+const USA = countries.find(country => country.value === "USA");
+const nonUSACountries = countries.filter(country => country.value !== "USA");
 
 const internationalAddressFields = {
   street: {
-    type: 'string',
+    type: "string",
     minLength: 1,
     maxLength: 52
   },
   street2: {
-    type: 'string',
+    type: "string",
     minLength: 1,
     maxLength: 52
   },
   street3: {
-    type: 'string',
+    type: "string",
     minLength: 1,
     maxLength: 52
   },
   city: {
-    type: 'string',
+    type: "string",
     minLength: 1,
     maxLength: 40
   },
   state: {
-    type: 'string',
+    type: "string",
     minLength: 1,
     maxLength: 20
   },
   postalCode: {
-    type: 'string',
+    type: "string",
     minLength: 1,
     maxLength: 30
   }
-}
+};
 
 const domesticSchoolAddress = {
-  required: ['street', 'city', 'state', 'country', 'postalCode'],
+  required: ["street", "city", "state", "country", "postalCode"],
   properties: {
     country: {
-      type: 'string', // TYPE: text (255)
-      'enum': [USA.label],
+      type: "string", // TYPE: text (255)
+      enum: [USA.label],
       default: USA.label
     },
     street: {
-      type: 'string',
+      type: "string",
       minLength: 1,
       maxLength: 84 // street + street2 + street3 length must be < 255
     },
     street2: {
-      type: 'string',
+      type: "string",
       minLength: 1,
       maxLength: 84 // street + street2 + street3 length must be < 255
     },
     street3: {
-      type: 'string',
+      type: "string",
       minLength: 1,
       maxLength: 84 // street + street2 + street3 length must be < 255
     },
     city: {
-      type: 'string',
+      type: "string",
       minLength: 1,
       maxLength: 255
     },
     state: {
-      type: 'string',
-      'enum': constants.states50AndDC.map(state => state.value), // backend accepts abbreviated state names
-      enumNames: constants.states50AndDC.map(state => state.label)
+      type: "string",
+      enum: constants.salesforceStates.USA.map(state => state.value), // backend accepts abbreviated state names
+      enumNames: constants.salesforceStates.USA.map(state => state.label)
```

Comment:
Here I was expecting constants.states.USA not salesForceStates.

Reply:
Updated it to constants.states.USA